Commands/Search.cs

```diff
@@ -0,0 +1,111 @@
+﻿
+//System
```

Comment:
We don't do these anywhere else in the project, probably best to stay consistent